pkg/storage/export/export/export_test.go

```diff
@@ -1392,15 +1392,17 @@ var _ = Describe("Export controller", func() {
 		pvc := createPVC("pvc", string(cdiv1.DataVolumeKubeVirt))
 		pvc.Spec.DataSource = &k8sv1.TypedLocalObjectReference{}
 		pvc.Spec.DataSourceRef = &k8sv1.TypedObjectReference{}
+		pvc.Spec.StorageClassName = pointer.P("somesc")
 		pvcInformer.GetStore().Add(pvc)
 		vm := createVMWithDVTemplateAndPVC()
 		dvs, err := controller.generateDataVolumesFromVm(vm)
 		Expect(err).ToNot(HaveOccurred())
 		Expect(dvs).To(HaveLen(1))
 		Expect(dvs[0]).ToNot(BeNil())
 		Expect(dvs[0].Name).To((Equal("pvc")))
-		Expect(dvs[0].Spec.PVC.DataSource).To(BeNil())
```

Comment:
Expect PVC to be nil?

Reply:
Good point! thanks

Reply:
how about adding some check in the storage field that you do copy from the pvc .. like volumemode?(will need to add it to the pvc first since it is not initiated)

Reply:
great idea, added